src/Files.App/Actions/FileSystem/CreateFolderWithSelectionAction.cs

```diff
@@ -18,6 +18,8 @@ public RichGlyph Glyph
 
 		public bool IsExecutable =>
 			context.ShellPage is not null &&
+			context.PageType != ContentPageTypes.RecycleBin &&
+			context.PageType != ContentPageTypes.ZipFolder &&
```

Comment:
There are probably other actions that need to be disabled in Recycle Bin.

Reply:
We should check it, but it's a separate issue.

Reply:
True